assets/js/googlesitekit/widgets/components/WidgetWrapper.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * WidgetWrapper component.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import { HIDDEN_CLASS } from '../util/constants';
+import { Cell } from '../../../material-components';
+
+const WidgetWrapper = ( { gridColumnWidth, children } ) => {
```

Comment:
@johnPhillips how about other sizes? What if the `gridColumnWidth` equals to 5 or 7? It will return undefined and the widget won't be displayed at all, right?

Reply:
Yeah, true. Looking at the code and tests I'm not sure 5 or 7 could happen, but I will handle this just in case.

Reply:
Note that I tried to keep the output (in terms of CSS classes in the snapshots and unit tests etc.) the same after this refactor as it was before, and there doesn't seem to be any pre-existing examples of 5 or 7 that I can see. I added these to be handled like `12` as I'm unsure how the mobile / tablet sizing would go in this scenario.